packages/core/helper-plugin/tests/server.js

```diff
@@ -0,0 +1,14 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+export const server = setupServer(
```

Comment:
This server setup seems a bit specific to the useRBAC hook. Do you think it's worth setting it up in `packages/core/helper-plugin/src/hooks/tests/useRBAC.test.js` instead?

Reply:
From previous FE syncs there's ambition you move the MSW servers to the root of their perspective projects, so this is the basis of this move. That way as our FE becomes more coherent we can share test data between tests ✌🏼